examples/image_embedding/EmbeddingModel_example_to_train_triplet_model_with_data_iterations.ipynb

```diff
@@ -4205,17 +4148,8 @@
     "_Feature"
    ],
    "window_display": false
-  },
-  "pycharm": {
-   "stem_cell": {
-    "cell_type": "raw",
-    "source": [],
-    "metadata": {
-     "collapsed": false
-    }
-   }
   }
  },
  "nbformat": 4,
- "nbformat_minor": 1
-}
\ No newline at end of file
+ "nbformat_minor": 4
+}
```

Comment:
EmbeddingModel_train_API_example_for_triplet_model.ipynb
similar comments from above (e.g., the overview section and pretty print of some cells)
+
we need more comments to guide people for [20] and [21].